blog-cse/2023-03-16-application-update.md

```diff
@@ -0,0 +1,28 @@
+---
+title: March 16, 2023 Application Update
+hide_table_of_contents: false
+keywords:
+  - sumo logic
+  - service release notes
+  - cloud siem
+  - cse
+tags: [cloud siem release notes, application update]
```

Comment:
Table of contents is shown by default, so you can remove the "false" declaration. For keywords, I'd make them specific to the release note (rather than generic 'sumo logic'). 'application update' tag is ok - makes it easier for users to sort CSE Release Notes.

```suggestion
keywords:
  - cloud siem
  - entity timeline
tags: [application update]
```